src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,53 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE = 10;
+    private final K[] key;
+    private final V[] value;
+    private int size = 0;
```

Comment:
class fields always have default values
```suggestion
    private int size;
```